features/serializer/group_filter_bis.feature

```diff
@@ -0,0 +1,442 @@
+Feature: Filter with serialization groups on items and collections (bis)
```

Comment:
Why (bis)?

Reply:
We have an issue on the ulimit again, this file has too many scenarios and need to be splited...

Reply:
Ok :(. Can we found a more descriptive name?

Reply:
`_2`? :laughing: